draft-ietf-ppm-dap.md

```diff
@@ -624,87 +622,77 @@ Helper:
-validity ({{Section 7.1 of !VDAF}}); the process of verifying this proof
-reveals nothing about the underlying measurement beyond its validity.
-
-The specific properties attested to by the proof vary depending on the
-measurement being taken. For instance, to measure the time the user took
-performing a given task the proof might demonstrate that the value reported was
-within a certain range (e.g., between `0` and `60` seconds). By contrast, to
-report which of a set of `N` options the user select, the report might contain
-`N` integers and the proof would demonstrate that `N-1` were `0` and the other
-was `1`.
-
-It is important to recognize that "validity" is distinct from "correctness".
-For instance, the user might have spent `30` seconds on a task but the Client
-might report `60` seconds. This is a problem with any measurement system and
-DAP does not attempt to address it; it merely ensures that the data is within
-acceptable limits, so the Client could not report `10^6`  or `-20` seconds.
+validity ({{Section 7.1 of !VDAF}}). Verifying this proof reveals nothing about
+the underlying measurement but its validity.
+
+The specific properties attested to by the proof depend on the measurement being
+taken. For instance, if the task is measuring the latency of some operation, the
+proof might demonstrate that the value reported was between 0 and 60 seconds.
+But to report which of `N` options a user selected, the report might contain `N`
+integers and the proof would demonstrate that `N-1` of them were `0` and the
+other was `1`.
+
+"Validity" is distinct from "correctness". For instance, the user might have
+spent `30` seconds on a task but might report `60` seconds. This is a problem
+with any measurement system and DAP does not attempt to address it. DAP merely
+ensures that the data is within the chosen limits, so the Client could not
+report `10^6`  or `-20` seconds.
 
 ## Replay Protection {#replay-protection}
 
 Another goal of DAP is to mitigate replay attacks in which a report is
-aggregated multiple times within a batch or across multiple batches. This would
+aggregated in multiple batches or multiple times in a single batch. This would
 allow the attacker to learn more information about the underlying measurement
 than it would otherwise.
 
 When a Client generates a report, it also generates a random nonce, called the
 "report ID". Each Aggregator is responsible for storing the IDs of reports it
-has aggregated for a given task. To check whether a report has been replayed,
-it checks whether the report's ID is in the set of stored IDs.
-
-Note that IDs do not need to be stored indefinitely. The protocol allows
-Aggregators to enforce replay only for a sliding time window (e.g., within the
-last two weeks of the current time) and reject reports that fall outside of the
-replay window. This allows implementation to save resources by forgetting old
-report IDs.
+has aggregated and rejecting replayed reports.
```

Comment:
Should we say `repeated` instead of `replayed`, to be more encompassing of errors rather than malicious action?

Reply:
I don't think "repeated" would be wrong, but we frequently discuss replay attacks and replayed reports elsewhere in the draft, so I think consistency is important.